### ISSUE TITLE
Enhancement: add 3D pickup animation for the dagger

### ISSUE THREAD

**Issue by aredfan** [LostArtefacts/TRX#1830]
In OG TR2, when Lara pulls the dagger from the dragon - there's no sprite displayed on the HUD.

**Comment by walkawayy** [comment]
@lahm86 this will also require #1634 I think (or I guess the inverse of that?). The dagger is `O_PUZZLE_OPTION_2` which is ID `179`. However, the game doesn't have a sprite for the dagger aka `O_PUZZLE_ITEM_2` which is ID `175`. So we could make this work for 3D pickups only at the moment.

**Comment by lahm86** [comment]
Yes, we would either have to come up with a sprite to inject (and work out where to put it) or only support this if 3D pickup HUD display is enabled.

**Comment by rr-** [comment]
> Yes, we would either have to come up with a sprite to inject (and work out where to put it) or only support this if 3D pickup HUD display is enabled.

IMO we should supply a sprite. Should be relatively easy to craft one since we already have the 3d model for it.

**Comment by aredfan** [comment]
Here's an attempt for the sprite although I'm no artist.

![Image](https://github.com/user-attachments/assets/c7664ce0-fa55-4f3d-8e6b-de9a1742a804)

**Comment by lahm86** [comment]
This looks fantastic!
